kallisto-compose.yaml

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2022 by the Lucyfer Contributors
+
+# vim: set fileencoding=utf-8:
+
+
+version: '3'
+
+services:
+    kallisto_service:
+        container_name: 'kallisto'
+        environment:
+            BLAZEGRAPH_UID: '${BLAZEGRAPH_UID}'
+            BLAZEGRAPH_GID: '${BLAZEGRAPH_GID}'
+        image: '${KALLISTO_IMAGE}'
+        ports:
+            - 8889:9999
```

Comment:
Maybe specifying the local host port via environment variable rather than hardcoding it here would be better?  This would allow easier customization of multiple environments running at the same time and avoid local host port collisions.